dashboard/modules/job/job_manager.py

```diff
@@ -269,7 +284,30 @@ def _get_current_node_resource_key(self) -> str:
                         return key
         else:
             raise ValueError(
-                "Cannot found the node dictionary for current node.")
+                "Cannot find the node dictionary for current node.")
+
+    def _handle_supervisor_started(self, job_id: str,
+                                   result: Optional[Exception]):
+        if result is None:
+            return
+        elif isinstance(result, RuntimeEnvSetupError):
+            logger.info(f"Failed to set up runtime_env for job {job_id}.")
```

Comment:
logger.error same as below

Reply:
ah I left this as info because it's a user-level error, not an error in the job submission code. does that make sense to you?